applications/asset_tracker_v2/src/modules/location_module.c

```diff
@@ -268,6 +268,10 @@ static void search_start(void)
 			DATA_FETCH_TIMEOUT_NEIGHBORHOOD_SEARCH * MSEC_PER_SEC;
 	}
 
+	if (methods_index_gnss != -1) {
+		config.methods[methods_index_gnss].gnss.priority_mode = true;
```

Comment:
And I want to confirm from @simensrostad that this is desired in ATv2. Read the description in location.h configuration description.

Reply:
@simensrostad Should we remove this line from ATv2 in NCS 2.2.0 context?

Reply:
Hi, yes I think we should. We need to test and figure out the possible shortcomings of enabling GNSS prio in the application before we can comfortably enable it.

Reply:
Fine, I'll remove it.